src/fileseq/frameset.py

```diff
@@ -80,15 +81,17 @@ def __new__(cls, *args, **kwargs):
         :raises: :class:`fileseq.exceptions.ParseException` if the frame range
                  (or a portion of it) could not be parsed
         """
-        self = super(cls, FrameSet).__new__(cls, *args, **kwargs)
+        self = super(cls, FrameSet).__new__(cls)
```

Comment:
I wasn't clear on the implications of removing the `*args, **kwargs` here, but it doesn't seem to impact the usage of direct calls to `__new__()` anyways. Guess they were unneeded the whole time.

Reply:
tbh I've never got into writing a custom __new__ and I'm not clear on this. I'm a bit worried that the py2/py3 behaviour _could_ diverge in a situation where __new__ can be called without __init__ being called but idk.

Reply:
Its probably fine. We use it internally to allocate an instance without calling the constructor, and then manually update its state. It's not intended for any kind of public usage. So we are probably fine here.